src/app/llms.txt/route.ts

```
@@ -1,49 +1,43 @@
+export function GET() {
+  const pages = source.getPages();
+
+  const bySection = new Map<string, typeof pages>();
+  for (const page of pages) {
+    const section = page.slugs[0] ?? 'other';
+    const list = bySection.get(section) ?? [];
+    list.push(page);
+    bySection.set(section, list);
+  }
+
+  const lines: string[] = ['# Novu Documentation', '', '> Notification infrastructure for developers.', ''];
```

ESLint

[error] Replace `'#·Novu·Documentation',·'',·'>·Notification·infrastructure·for·developers.',·''` with `⏎····'#·Novu·Documentation',⏎····'',⏎····'>·Notification·infrastructure·for·developers.',⏎····'',⏎··`

(prettier/prettier)